src/Enterspeed.Source.Sdk/Domain/Connection/EnterspeedConnection.cs

```diff
@@ -59,6 +60,10 @@ private void Connect()
             _httpClientConnection.DefaultRequestHeaders.Add("X-Api-Key", ApiKey);
             _httpClientConnection.DefaultRequestHeaders.Add("Accept", "application/json");
 
+            #if NETSTANDARD2_0_OR_GREATER || NET || NETCOREAPP2_0_OR_GREATER
```

Comment:
I've left out older netstandard versions, since they do not support such a simple way of retrieving assembly version